beacon-chain/core/epoch/precompute/type.go

```diff
@@ -24,7 +24,7 @@ type Validator struct {
 	IsPrevEpochSourceAttester bool
 	// IsPrevEpochTargetAttester is true if the validator attested previous epoch target.
 	IsPrevEpochTargetAttester bool
-	// IsHeadAttester is true if the validator attested head.
+	// IsPrevEpochHeadAttester is true if the validator attested head.
```

Comment:
Thanks, you should change the comment to reflect it too
```suggestion
	// IsPrevEpochHeadAttester is true if the validator attested the previous epoch head.
```

Reply:
Thanks. Modified